cpio-2.13-mutiple-definition.patch

```diff
@@ -0,0 +1,605 @@
+
```

Comment:
Not sure how this html file is any useful?

Reply:
It is not, my bad when "Saving link as".

Reply:
For the moment I am not merging this. It seems it is not opening rpm files anyways, at least it does not in my machine.